spec/unit/pops/lookup/interpolation_spec.rb

```diff
@@ -161,6 +161,10 @@ def expect_lookup(*keys)
       expect(interpolator.interpolate('a dot e: %{a.d}', lookup_invocation, true)).to eq('a dot e: (scope) a dot d is a hash entry')
     end
 
+    it 'should report a key missing and replace with empty string when a dotted key is used to navigate into a structure and then not found' do
+      expect(interpolator.interpolate('a dot n: %{a.n}', lookup_invocation, true)).to eq('a dot n: ')
+    end
```

Comment:
It tests if the resulting string contains nothing (the empty string) at the place where the interpolation took place. The title could perhaps be more explicit. The sub-lookup "reports the key missing" and the interpolation reacts to that by inserting the empty string. Then again, it's all under the category "when using dotted keys".